biztalk/install-and-config-guides/configure-biztalk-server.md

```diff
@@ -243,6 +243,32 @@ BAM alerts require BAM tools to be enabled.
     |Windows group|You can keep the default group name, or enter something custom. |
     |Windows SharePoint Services Adapter Web site|Select the Web site that hosts the Windows SharePoint Service Adapter web service.|
 
+### Configure the REST APIs
+>[NOTE]
+>Start from BizTalk Server 2016 feature pack, you can use REST APIs to manage the different artifacts in your BizTalk Server enviroments and operational data flowing through. 
+
+1. Select **REST APIs**.
+2. Configure the following:
+
+    |Use this|To do this|
+    | --- | --- |
+    |Enable BizTalk REST APIs | Check this if you want enable BizTalk REST APIs. | 
+    |Web service accounts | Specify the account used to run the IIS services. <ul><li>Management Service: REST APIs for BizTalk Server artifacts management. The default URI for **Management Service** APIs is http(s)://localhost/BizTalkManagementService. </li><li>Operational Data Service: Service providing information on the instances and messages flowing through your BizTalk Server environment. The operational data feed is the same data you get looking at Group Hub in BizTalk Server Administration.The default URI for **Operational Data service** APIs is http(s)://localhost/BizTalkOperationalDataService.</li></ul>|
+    |Windows groups | BizTalk REST APIs use Windows authentication, specify the Windows groups allowed to access the APIs here. <ul><li>Management REST APIs User Role: Specify this as an extra layer of security check or keep default, because actually windows user who *request* (not user hosting the service) to management REST APIs will take the access challenge. </li><li>Operational Data REST APIs User Role: Specify user group allowed to access.</li>|
+    |REST APIs website|Select the Web site to host the both the REST APIs. |
+
+### Configure BizTalk TMS
+
+>[NOTE]
+>"Enterprise Single Sign-On Service"(SSO) is major dependency for TMS, TMS should be configured on server which has SSO service running. Also TMS is recommanded to configure only on one server in a BizTalk Group.
```

Comment:
suggestion:
> [!IMPORTANT]
> - TMS should be configured on a server which also has the "Enterprise Single Sign-On Service" (ENTSSO) running.
> - TMS should only be configured on one server in a BizTalk Group. For high-availability scenarios, you may configure TMS on multiple servers in an Active-Passive topology

Reply:
I don't think you need to mention: "Enterprise Single Sign-On Service"(SSO) is major dependency for TMS

Reply:
Agree, and removed it.

Reply:
SSO service dependency should be marked as important, not just a note.
Also, please add the information about high-availability scenarios

Reply:
FIxed.